bigtable/bigtable.go

```diff
@@ -397,6 +400,7 @@ func (t *Table) readRows(ctx context.Context, arg RowSet, f func(Row) bool, mt *
 		*headerMD, _ = stream.Header()
 		for {
 			res, err := stream.Recv()
+			mt.currOp.setFirstRespTime(time.Now())
```

Comment:
What happens when multiple responses are received?

what happens when there are no rows returned? 

is the behavior consistent with java?

Does this work across retries?